controllers/test/data/crd/strimzi-crds-0.30.0.yaml

```diff
@@ -9407,7 +9407,7 @@ spec:
                     x-kubernetes-preserve-unknown-fields: true
                     type: object
                 description: CPU and memory resources to reserve.
-              whitelist:
+              allowlist:
```

Comment:
I'm not sure if these CRDs are still used but did you validate Allowlist works with strimzi?